src/lf_apply.rs

```diff
@@ -43,8 +44,9 @@ unsafe fn backup_lpf<BD: BitDepth>(
     };
     // The first stripe of the frame is shorter by 8 luma pixel rows.
     let mut stripe_h = ((64 as c_int) << (cdef_backup & sb128)) - 8 * (row == 0) as c_int >> ss_ver;
-    src = src.offset((stripe_h - 2) as isize * BD::pxstride(src_stride as usize) as isize);
-    if c.n_tc == 1 as c_uint {
+    src_offset = src_offset
+        .wrapping_add_signed((stripe_h - 2) as isize * BD::pxstride(src_stride as usize) as isize);
```

Comment:
Why do we need wrapping here?  These are offsets; they shouldn't get anywhere near `usize::MAX`, right?